### PR TITLE
ChartDetails Updates

### PR DESCRIPTION
**Pull request by rayhm6419**
**1. fixed the text bug**
Before:
image

After:
image

**2.footer logo**
Before:
image

After:
image

**3.Added Background color to WorkingOn Section**
image

4. Cleaned some unused CSS code in ChartView.css

### CODE REVIEW

File: src/components/footer/Footer.css [hunk 1]
```diff
@@ -43,11 +43,13 @@ footer {
     width: 60px;
     height: 60px;
     padding-right: 0.4rem;
+    border-radius: 0;
 }
 
 .footer-text-logo {
     width: 48px;
     height: 48px;
+    border-radius: 0;
```
Comment (lima01) [inline src/components/footer/Footer.css:52]: First of all, this change is not related to the PR.
Secondly, the fix should be in InterviewPage.css. Need to remove the corder-radius from img and move it to interview-profile.

**Review by lima01 — CHANGES_REQUESTED** [review]

File: src/components/footer/Footer.css (hunk 1 above)
Comment (rayhm6419) [reply src/components/footer/Footer.css]: I removed and will do it in next PR

File: src/components/footer/Footer.css [hunk 2]
```diff
@@ -43,6 +43,7 @@ footer {
     width: 60px;
     height: 60px;
     padding-right: 0.4rem;
+    border-radius: 0;
```
Comment (lima01) [inline src/components/footer/Footer.css:46]: This is not needed if you fix in InterviewPage.css

Comment (rayhm6419) [reply src/components/footer/Footer.css]: I removed 'border-radius' from the footer.css and set 'border-radius' of all images to 0. Now all the images would not affected by the border-radius as before.

And only apply border-radius:50% for the interview profile img to make it round 
image
image

**Comment by jiayingxie** [comment]
Could you please resolve the conflicts?
